tensorboard/components/tf_data_selector/tf-data-select-row.ts

```diff
@@ -20,7 +20,6 @@ enum Type {
 }
 
 const MAX_RUNS_TO_ENABLE_BY_DEFAULT = 20;
-const STORAGE_ALL_VALUE = '$all';
 const STORAGE_NONE_VALUE = '$none';
```

Comment:
Does it perhaps make sense to just move this over too, so they're kept side-by-side?